lib/dependencies/ContextDependencyHelpers.js

```diff
@@ -45,7 +45,9 @@ ContextDependencyHelpers.create = (
 	Dep,
 	range,
 	param,
+	paramExpr,
 	expr,
+	parser,
```

Comment:
To avoid a breaking change, make this argument optional. In this case expressions are not walked.

Reply:
OK, thanks! Then in this PR I'll only handle `import()` case, then later working on AMD `define`, `require`, CommonJS `require` and `require.resolve`